experimental/Schemes/IdealSheaves.jl

```diff
@@ -1842,4 +1882,82 @@ function sub(I::AbsIdealSheaf)
   return domain(inc), inc
 end
 
+########################################################################
+# Radicals of ideal sheaves                                            #
+########################################################################
+
+underlying_presheaf(rad::RadicalOfIdealSheaf) = rad.Ipre
+original_ideal_sheaf(rad::RadicalOfIdealSheaf) = rad.orig
+
+function produce_object(rad::RadicalOfIdealSheaf, U::AbsAffineScheme)
+  result = radical(original_ideal_sheaf(rad)(U))
+  set_attribute!(result, :is_radical=>true) # Necessary? Or should the radical computation take care of this?
+  return result
+end
+
+function is_subset(I::AbsIdealSheaf, rad::RadicalOfIdealSheaf)
+  X = scheme(rad)
+  @assert X === scheme(I) "ideal sheaves do not live on the same scheme"
+  for U in affine_charts(X)
+    all(g->radical_membership(g, rad(U)), gens(I(U))) && return false
+  end
+  return true
+end
+
+function is_subset(I::PrimeIdealSheafFromChart, rad::RadicalOfIdealSheaf)
+  X = scheme(rad)
+  @assert X === scheme(I) "ideal sheaves do not live on the same scheme"
+  U = original_chart(I)
+  return all(g->radical_membership(g, rad(U)), gens(I(U)))
+end
+
+function radical(I::AbsIdealSheaf)
+  result = RadicalOfIdealSheaf(I)
+  set_attribute!(result, :is_radical=>true) # Some methods might be blind to is_radical and only want to check `is_known_to_be_radical` via attributes. Setting this makes sure they get it. 
```

Comment:
the safer way is probably to do this in the constructor of `RadicalOfIdealSheaf` ?

Reply:
But this is the constructor, isn't it?

Reply:
it is the outer constructor. I would but it in the inner constructor `RadicalOfIdealSheaf`